modules/_labs/color-picker/react/lib/ColorPicker.tsx

```diff
@@ -94,6 +94,10 @@ const defaultColors = [
   colors.blackPepper400,
 ];
 
+const ColorPickerContainer = styled('div')({
+  width: 216,
```

Comment:
Not sure how we want to handle this, at this set width the columns align perfectly, should we make this a prop or add an explanation in the code as to why it;s this set width?